bamliquidator_internal/makefile

```diff
@@ -137,3 +140,11 @@ clean:
 
 install: all
 	install $(EXECUTABLES) $(DESTDIR)$(bindir)
+
+docker:
+	docker build . --build-arg GIT_COMMIT=$(GIT_COMMIT) -t bioliquidator/bamliquidator:latest -t bioliquidator/bamliquidator:$(VERSION)
+
+docker-hub: docker
```

Comment:
Would you want this to be done automated? Docker has user tokens, but not user/repo specific ones, so you'd give access to your whole account. If you want to generate a CI recipe that has a repo specific token, I'd check out quay.io - I've been using them since Docker was hacked and am very happy.

Reply:
I would support someone else automating, but I'm not working on that right now